perfkitbenchmarker/linux_benchmarks/mysql_service_benchmark.py

```diff
@@ -131,6 +131,15 @@
 SECONDS_UNIT = 'seconds'
 MS_UNIT = 'milliseconds'
 
+MYSQL_SVC_OLTP_TABLES_COUNT = 'mysql_svc_oltp_tables_count'
```

Comment:
moving just those necessary variables will do.

The benefit of doing so is we ensure the name of the flag will always be the same as the labels we put into metadata. We will never misspell.